README.md

```diff
@@ -108,6 +109,16 @@ To test secondary storage access, we are going to:
 
 ## More information
 
-- [About Azure Storage Accounts](https://docs.microsoft.com/azure/storage/storage-create-storage-account)
-- [Designing HA Apps with RA-GRS Storage](https://docs.microsoft.com/azure/storage/common/storage-designing-ha-apps-with-ragrs)
-- [Azure Storage Replication](https://docs.microsoft.com/azure/storage/storage-redundancy)
+- [About Azure Storage Accounts]
```

Comment:
I like this link structure, but if you are going to do here, then you should do for all samples.